### ISSUE TITLE
Release questions

### ISSUE THREAD

**Issue by rwbc** [Checkmate6659/black-cat-chess#1]
Hi,

I have compiled your program successfully (WIN) for several versions and my question is, if you want to remain anonymous?
Ofc that's completely up to you, but just in case you consider to have it released under a real name I would have added it
to the XB/UCI chronology. (anonymous' are excluded there for several reasons)

https://docs.google.com/spreadsheets/d/1GXoy3lGhdKPJbatdw2_2dhWT3TkpthPcNtHs51MQydM
https://docs.google.com/spreadsheets/d/1KWIlXYvEnACeFc4XP3N63pyBSHCG4RqVSCAH22QR1l0

Thanks for reading and good luck with further progress on your program.

Guenther Simon (RWBC)

**Comment by Checkmate6659** [comment]
Hi Simon,

My name is Alexander Wimmer. How did you compile it with windows? I 
tried using the windows compiling command in my makefile with clang but 
I got an error. Also, since you sent me this email, I improved my engine 
a fair bit. I don't know its strength yet, but I'm planning on measuring 
it and releasing a new version as soon as possible.
Btw, this is my commercial/spam email box that I use to register to 
basically every website (which is also part of the reason why I took 
like 3 weeks to respond). My "real" email is ***@***.***

Alexander Wimmer

Le 2022-09-21 10:23, rwbc a écrit :

**Comment by rwbc** [comment]
Hi Alexander,

Sorry for the late answer, I was away from computerchess and very busy for some months.

My compiling environment uses msys2, which has its own posix implementation integrated,
all versions compile just fine here, by removing posix.cpp from the github sources.

I offer my WIN compilations now from my site for download. If you want to add them to your releases,
you are of course free to do so.
(Note that I did not compile 2.4 because it uses a SF network)

BTW from your last line in the reply to my issue I see you are localized in a French speaking country?
For a proper entry in the XB/UCI chronology I would need your nationality, for now I go with France.

Guenther

**Comment by Checkmate6659** [comment]
Hi Guenther,

I am indeed from France. Also, if a SF network isn't allowed, I may add a fork later that uses a smaller home-trained net instead of the old SF net.

Alexander
